src/tools/clippy/clippy_lints/src/loops/never_loop.rs

```diff
@@ -206,6 +206,12 @@ fn never_loop_expr(expr: &Expr<'_>, ignore_ids: &mut Vec<HirId>, main_loop_id: H
                 NeverLoopResult::AlwaysBreak,
             )
         }),
+        ExprKind::Become(e) => {
```

Comment:
this can be grouped into the previous arm

Reply:
no, `e` in the previous arm is an option